frontend/e2e-test/test/e2e/pages/enduser/enduser-navigation.ts

```diff
@@ -132,6 +132,26 @@ export default class EnduserPage {
         `attachment;filename=${decisionName}_Jari-Petteri_Mukkelis-Makkelis_Vetelä-Viljami_Eelis-Juhani_Karhula.pdf`
       )
   }
+
+  readonly urgentAttachmentsUpload = Selector(
+    '[data-qa="file-upload-urgent-attachments"]'
+  )
+
+  async uploadUrgentFile(file: string) {
+    await t.setFilesToUpload(
+      this.urgentAttachmentsUpload.find('[data-qa="btn-upload-file"]'),
+      [file]
+    )
+  }
+
+  async assertUrgentFileHasBeenUploaded(file: string) {
+    await t
+      .expect(
+        this.urgentAttachmentsUpload.find('[data-qa="files"]').withText(file)
+          .exists
+      )
+      .ok()
```

Comment:
I think doing something like adding a `data-qa="upload-status-done"` to files that have been fully uploaded and then asserting that the file has that property would be a good idea here.

Reply:
Uploaded file has now "uploaded" class and test uses it